internal/converter/internal/staticconvert/internal/build/builder_traces.go

```diff
@@ -0,0 +1,103 @@
+package build
+
+import (
+	"fmt"
+	"reflect"
+
+	"github.com/grafana/agent/internal/converter/diag"
+	"github.com/grafana/agent/internal/converter/internal/otelcolconvert"
+	"github.com/grafana/agent/internal/static/traces"
+	otel_component "go.opentelemetry.io/collector/component"
+	"go.opentelemetry.io/collector/config/configtelemetry"
+	"go.opentelemetry.io/collector/exporter/loggingexporter"
+	"go.opentelemetry.io/collector/otelcol"
+	"go.uber.org/zap/zapcore"
+)
+
+func (b *ConfigBuilder) appendTraces() {
+	if reflect.DeepEqual(b.cfg.Traces, traces.Config{}) {
+		return
+	}
+
+	for _, cfg := range b.cfg.Traces.Configs {
+		otelCfg, err := cfg.OtelConfig()
+		if err != nil {
+			b.diags.Add(diag.SeverityLevelCritical, fmt.Sprintf("failed to load otelConfig from agent traces config: %s", err))
+			continue
+		}
+
+		removeReceiver(otelCfg, "traces", "push_receiver")
+		b.translateAutomaticLogging(otelCfg, cfg)
+
+		// Let's only prefix things if we are doing more than 1 trace config
+		labelPrefix := ""
+		if len(b.cfg.Traces.Configs) > 1 {
+			labelPrefix = cfg.Name
+		}
+		b.diags.AddAll(otelcolconvert.AppendConfig(b.f, otelCfg, labelPrefix))
+	}
+}
+
+func (b *ConfigBuilder) translateAutomaticLogging(otelCfg *otelcol.Config, cfg traces.InstanceConfig) {
+	if _, ok := otelCfg.Processors[otel_component.NewID("automatic_logging")]; !ok {
+		return
+	}
+
+	if cfg.AutomaticLogging.Backend == "stdout" {
+		b.diags.Add(diag.SeverityLevelWarn, "automatic_logging for traces has no direct flow equivalent. "+
+			"A best effort translation has been made to otelcol.exporter.logging but the behavior will differ.")
+	} else {
+		b.diags.Add(diag.SeverityLevelError, "automatic_logging for traces has no direct flow equivalent. "+
+			"A best effort translation can be made for outputting to console instead by bypassing errors.")
```

Comment:
Can it be made more apparent here that the best-effort translation can only log to stdout and not directly to a loki instance as before?